packages/cashier/src/stores/general-store.js

```diff
@@ -11,6 +11,7 @@ export default class GeneralStore extends BaseStore {
         super({ root_store });
 
         makeObservable(this, {
+            advertiser_info: observable,
```

Comment:
@farrah-deriv Is it possible to move this into P2P stores?

Reply:
Will check @farzin-deriv